extIds.txt

```diff
@@ -1,4 +1,3 @@
-8btxhs
```

Comment:
hmm, shouldn't the .gitignore prevent this from being committed? In any case this isn't meant to be modified.